docs/user_manual/hosting_a_game_server.md

```diff
@@ -12,7 +12,11 @@ connect to the replacement login server, follow these steps:
    It contains `msvcp140.dll`, which is needed by TAMods-Server and the UDP proxy used by the
    firewall script.
 
-4. Install python 3
+4. Install python 3.
+
+   Make sure you check the `Add Python to PATH` checkbox (see image). Otherwise you will have some issues with the next steps.
```

Comment:
Where will they have problems? I think if we only specify "add to path" we will need to make sure that each command line starts with "python". If we don't want that then we need to make sure they install the python launcher too, which allows one to start python scripts directly via file association.

Reply:
I understand what you mean. 
I think it would just be simpler to add to PATH and then add python to the start of all the commands. But its simpler to us, not really to a beginner user.
I could add the note to install python launcher instead, like you are suggesting. 
You decide what would be best.

Reply:
I think the pip install command will not work, but you added a way around that, so i guess i can remove the `Otherwise you will have some issues with the next steps.` line

Reply:
What about this:

4. Install python 3.

It is recommended during installation to include the py launcher (to be able to start scripts 
without running them as `python <scriptname>`) and to have python added to the PATH 
environment variable (so you can start python and pip without having to type the full path 
to the executables). If you don't do this then remember to specify `python.exe <scriptname>` 
or even something like `C:\Program Files (x86)\Python36\python <scriptname>` whenever the 
instructions tell you to run a script.

Reply:
added

Reply:
and formatted